github/requesting_review.md

```diff
@@ -0,0 +1,58 @@
+# Requesting review
+
+This document tries to explain who should be requested for review when crafting a PR.
+
+## Default
+
+By default a review **should** be requested from:
+
+- A team. Usually your team or the [team owning the component][ownership].
+- Individual stakeholders or person who can bring valuable input.
+
+By default a review **should not** be requested from:
+
+- Chapter or SIG.
```

Comment:
```suggestion
- Chapter or SIG. Exceptions are `happa` and `giantswarm-js-client`. PRs in these projects should always be reviewed by chapter FE.
```

Reply:
Can we maybe add that to `Exceptions` section?

Reply:
ok